### DISCUSSION TITLE
Hitachi Vantara UI Kit Issues With React Markdown

### DISCUSSION THREAD

**Discussion by prajwalkumar9** [lumada-design/hv-uikit-react#2791] [Q&A] [answered]
In our project we are using the latest version of react-markdown 
https://www.npmjs.com/package/react-markdown

But when we are using the Hitachi Vantara React UI Kit Lab Component for Table (https://lumada-design.github.io/uikit/master/?path=/docs/lab-table--main) its giving issues on deployment and the pages are not loading and resulting in errors in console.

Can you guys please check on this integration and help us in resolving this issue. It is working fine with the old react markdown package version 4.3.1 but issue persists in all the latest version of react markdown (Tried 8.0.3, 8.0.2, 8.0.1)

**Comment by francisco-guilherme** [comment, marked as answer]
Please take a look at the [issue](https://github.com/lumada-design/hv-uikit-react/issues/2800#issuecomment-1163323764) comment.
